blocks/hero/hero.js

```diff
@@ -20,7 +20,7 @@ function template(info) {
                 </div>
                 <div class="general-article-stage__details text--eyebrow">
                     ${info.Date}
-                    · ${info.Duration} Lesedauer
+                    · ${info.Duration}
```

Comment:
Please make the changes against the `simplify` branch. The markup for hero block has been completely changed (and simplified) there.